sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -5380,6 +5481,48 @@ namespace Azure { namespace Storage { namespace Blobs {
         return DeleteCreateResponse(context, std::move(pHttpResponse));
       }
 
+      struct SetBlobExpiryOptions
+      {
+        Azure::Core::Nullable<int32_t> Timeout;
+        ScheduleBlobExpiryOriginType ExpiryOrigin;
+        Azure::Core::Nullable<std::string> ExpiryTime;
+      }; // struct SetBlobExpiryOptions
+
+      static Azure::Core::Response<SetBlobExpiryResult> ScheduleDeletion(
+          const Azure::Core::Context& context,
+          Azure::Core::Http::HttpPipeline& pipeline,
+          const Azure::Core::Http::Url& url,
+          const SetBlobExpiryOptions& options)
+      {
+        unused(options);
```

Comment:
Remove `unused()` function.
Update this to (void) options.
Think we agreed about this on some of the team meetings?

Reply:
I didn't remember we have such an agreement

Reply:
Didn't find the prev chat about it.  Started a new one here:

https://teams.microsoft.com/l/message/19:a5180a2de11a40cb83a822f588d60c15@thread.skype/1600274130341?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1600274130341&teamName=Azure%20SDK&channelName=Language%20-%20C%E1%90%A9%E1%90%A9&createdTime=1600274130341

Seems like `(void)arg` is what most of people like